x/posts/internal/types/post.go

```diff
@@ -57,7 +57,7 @@ func (id *PostID) UnmarshalJSON(data []byte) error {
 }
 
 // ParsePostID returns the PostID represented inside the provided
-// value, or an error if no id could be parsed properly
+// value, or an error if no newID could be parsed properly
```

Comment:
I think this is a refactoring error

```suggestion
// value, or an error if no id could be parsed properly
```